ovn-tester/ovn_sandbox.py

```diff
@@ -59,77 +59,17 @@ def run(self, cmd="", stdout=None, raise_on_error=False):
 class Sandbox(object):
     def __init__(self, phys_node, container):
         self.phys_node = phys_node
-        self.container = container
-        self.channel = None
-
-    def ensure_channel(self):
-        if self.channel:
-            return
-
-        self.channel = self.phys_node.ssh.ssh.invoke_shell(width=10000,
-                                                           height=10000)
-        # Fail if command didn't finish after 1 minute.
-        self.channel.settimeout(60.0)
-        if self.container:
-            dcmd = 'docker exec -it ' + self.container + ' bash'
-            self.channel.sendall(f"{dcmd}\n".encode())
+        self._set_container_pid(container)
 
+    def _set_container_pid(self, container):
         stdout = StringIO()
-        # Checking + consuming all the unwanted output from the shell.
-        self.run(cmd="echo Hello", stdout=stdout, raise_on_error=True)
+        cmd = f' docker inspect -f \'{{.State.Pid}}\' {container}'
```

Comment:
This should be:
`cmd = 'docker inspect -f {{.State.Pid}} ' + f'{container}'`
Otherwise, f-string eats the `{}` and command returns not what was expected.